packages/notifications/src/Notification.php

```diff
@@ -0,0 +1,170 @@
+<?php
+
+namespace Filament\Notifications;
+
+use Filament\Support\Components\ViewComponent;
+use Livewire\Wireable;
+
+class Notification extends ViewComponent implements Wireable
+{
+    protected string $view = 'notifications::components.notification';
+
+    public string $id;
+
+    public bool $closed = false;
+
+    public string $title;
+
+    public ?string $description = null;
+
+    public ?string $icon = null;
+
+    public ?string $status = null;
+
+    public function __construct()
+    {
+        $this->id(uniqid());
```

Comment:
How about `Str::uuid()`?

Reply:
That's what I opted for myself, until I found the current implementation used `uniqid()`. 😂

3dcedd0cf061748f32e6cb6be6c0fec812e9d8fe